documentation/how-to/enable-superset-security-features.md

```diff
@@ -0,0 +1,128 @@
+# Enable security features
+This guide describes the implementation of security features such as encryption, authentication and HTML sanitization.
+
+## Terminate TLS at ingress
+Superset can terminate Transport Layer Security (TLS) at the ingress by leveraging the [Nginx Ingress Integrator Charm](https://charmhub.io/nginx-ingress-integrator).
+
+Deploy this by running:
+
+```bash
+juju deploy nginx-ingress-integrator --trust
+```
+
+### Using K8s secrets
+You can use a self-signed or production-grade TLS certificate stored in a Kubernetes secret. The secret is then associated with the ingress to encrypt traffic between clients and the Superset User Interface (UI).
+
+For self-signed certificates you can do the following:s
```

Comment:
question: Is there a special purpose for `following:s` or is it a typo?

Reply:
This was a typo, thank you, I've fixed it.